diff -ipuwr -x config.log -x config.status -x config.cache -x '*.Po' -x '*.pot' -x '*.bak' -x '*.orig' -x '*.rej' -x '*.new' -x '*~' -x '*.o' -x '*.exe' -x patches -x '*.dll' -x'*.la' -x res -x debian -x '*.log' -x '*.lib' -x'*.a' -x .libs -x'*.def' -x'*.uu' -x'*.rc' -x'*.ico' -x'*.lo' fax2png-1.0-orig/Makefile fax2png-1.0/Makefile
--- fax2png-1.0-orig/Makefile	2004-01-28 00:52:12.000000000 +0100
+++ fax2png-1.0/Makefile	2006-03-28 10:51:37.203125000 +0200
@@ -1,8 +1,24 @@
-CFLAGS=-O -pedantic
+CFLAGS=-D__GW32__ -D_LARGEFILE_SOURCE=1 -D_LARGEFILE64_SOURCE=1 -D_FILE_OFFSET_BITS=64 \
+	-Wall -O3 -fms-extensions -mms-bitfields  -fno-exceptions -fomit-frame-pointer \
+	-march=i386 -mcpu=i686 \
+	-idirafter D:/progra~1/gnuwin32/include -idirafter G:/glibc/include
+LDFLAGS = -Wl,-s -Wl,--force-exe-suffix -Wl,--enable-auto-import -Wl,--enable-runtime-pseudo-reloc \
+	-LD:/progra~1/gnuwin32/lib -lgw32c
+EXEEXT = .exe
 
-fax2png: fax2png.o
-	gcc fax2png.o -o fax2png -lpng -lz -ltiff 
+.SUFFIXES:
+.SUFFIXES: .c .o .exe .rc
 
-install: fax2png
-	install fax2png /usr/local/bin
+# Rule to make compiled resource (Windows)
+.rc.o:
+	windres --include-dir $(resdir) -i $< -o $@
+
+fax2png$(EXEEXT): fax2png.o fax2png-res.o
+	gcc fax2png.o  fax2png-res.o -o $@ $(LDFLAGS) -lpng -lz -ltiff 
+
+install: fax2png$(EXEEXT)
+	/bin/install.exe -cpD $< c:/progra~1/Fax2Png/bin/$<
+
+clean:
+	rm -f fax2png$(EXEEXT) fax2png.o
 
diff -ipuwr -x config.log -x config.status -x config.cache -x '*.Po' -x '*.pot' -x '*.bak' -x '*.orig' -x '*.rej' -x '*.new' -x '*~' -x '*.o' -x '*.exe' -x patches -x '*.dll' -x'*.la' -x res -x debian -x '*.log' -x '*.lib' -x'*.a' -x .libs -x'*.def' -x'*.uu' -x'*.rc' -x'*.ico' -x'*.lo' fax2png-1.0-orig/fax2png.c fax2png-1.0/fax2png.c
--- fax2png-1.0-orig/fax2png.c	2004-01-28 00:51:14.000000000 +0100
+++ fax2png-1.0/fax2png.c	2006-03-28 10:50:50.218750000 +0200
@@ -175,6 +175,7 @@ int main(int argc, char *argv[])
 		}
 	} else {
 		out = stdout;
+		SET_BINARY(STDOUT_FILENO);		
 	}
 	png_init_io(png_ptr, out);
 	/* Turning off filtering yields a large speed improvement at a 
